compiler/rustc_session/src/options.rs

```diff
@@ -670,6 +671,18 @@ mod parse {
         }
     }
 
+    pub(crate) fn parse_frame_pointer(slot: &mut FramePointer, v: Option<&str>) -> bool {
```

Comment:
Maybe `parse_force_frame_pointers` so that it matches the actual option?

Reply:
The other parse options reflect the value they parse.